sources/packages/backend/apps/db-migrations/src/sql/CASInvoices/Create-cas-invoices.sql

```diff
@@ -0,0 +1,43 @@
+CREATE TABLE sims.cas_invoices(
+  id SERIAL PRIMARY KEY,
+  cas_invoice_batch_id INT NOT NULL REFERENCES sims.cas_invoice_batches(id),
+  disbursement_receipt_id INT NOT NULL REFERENCES sims.disbursement_receipts(id),
+  cas_supplier_id INT NOT NULL REFERENCES sims.cas_suppliers(id),
+  invoice_number VARCHAR(40) NOT NULL,
+  invoice_status sims.cas_invoice_status NOT NULL,
+  invoice_status_updated_on TIMESTAMP WITH TIME ZONE NOT NULL,
```

Comment:
Minor. I believe the `invoice status updated by `is not added considering that the status will be updated on by system. But will it be worth considering now(expecting a manual update) or when requirement comes?

Reply:
Since the initial status would be pending, the idea was to keep both updated upon record creation.